check.go

```diff
@@ -287,4 +298,24 @@ func init() {
 		}
 		return nil
 	})
+
+	// Scan neulevel-1.0 extension
+	path = "epp > response > extension > " + ExtNeulevel10 + " extension > unspec"
+	scanResponse.MustHandleCharData(path, func(c *xx.Context) error {
+		dcd := &c.Value.(*response_).DomainCheckResponse
+		check := &dcd.Checks[len(dcd.Checks)-1]
```

Comment:
May also want to check `len(dcd.Checks)` is at least 1, since negative indexes are invalid. Is `dcd` short for `DomainCheckResponse`?